requirements/dev.txt

```diff
@@ -168,6 +169,10 @@ pre-commit==2.4.0 \
     --hash=sha256:5559e09afcac7808933951ffaf4ff9aac524f31efbc3f24d021540b6c579813c \
     --hash=sha256:703e2e34cbe0eedb0d319eff9f7b83e2022bb5a3ab5289a6a8841441076514d0 \
     # via -r dev.in
+prometheus-client==0.8.0 \
```

Comment:
this is an interesting entry.

Reply:
Looks like it's pulled in from the flower 0.9.4 -> 0.9.5 update